conf/defaults.ini

```diff
@@ -1013,6 +1013,12 @@ reset_basic_roles = false
 # Validate permissions' action and scope on role creation and update
 permission_validation_enabled = true
 
+# List of resources to generate managed permissions for upon creation (dashboard, folder, service-account, datasource)
+managed_permissions_on_creation = dashboard, folder, service-account, datasource
```

Comment:
I wonder if it would be better to have a `skip_managed_permissions_on_creation` - so there would be no change if this config option is not set. Any thoughts?

Reply:
I think the naming of these config properties is currently a bit confusing. I think the way I would think about this is:

```
set_managed_permissions = true
managed_permissions_resources = dashboard, folder, service-account, datasource (by default)
managed_permissions_seed_resources = dashboard, folder, service-account, datasource (by default)

```

Reply:
Thanks for the feedback! I've changed the config option names to `resources_with_managed_permissions_on_creation` (previously `managed_permissions_on_creation`) and `resources_with_seeded_wildcard_access` (previously `managed_permissions_wildcard_seeds_resources`). Does that sound better? I think adding a `set_managed_permissions` option might be a bit confusing (what would the expected behaviour be if `set_managed_permissions` is set to `false`, but not all resources are specified in `resources_with_managed_permissions_on_creation`?)